src/phase0/networking.rs

```diff
@@ -0,0 +1,29 @@
+use crate::primitives::Epoch;
+use enr;
+pub use multiaddr::Multiaddr;
+#[cfg(feature = "serde")]
+use ssz_rs::prelude::Bitvector;
+use std::time::Duration;
```

Comment:
whitespace after imports, I *think* `rustfmt` will do this for you...

Reply:
Sure, I thought this was on purpose. Apparently if there is already some whitespace `rustfmt` will default to a single whitespace. But if there is none, it won't do anything (tried with `const` and `fn` after imports).

Reply:
no, it was an omission for sure, thanks for adding it!